functions/src/definitions/common/sendWhatsappMessage.ts

```diff
@@ -278,7 +278,7 @@ async function sendWhatsappContactMessage(
     to: string
     type: string
     contacts: {
-      name: string
```

Comment:
Nice catch! Would this have thrown an error earlier?

Reply:
I actually don't think it affected runtime since the generated javascript code will use the intended shape instead of just string.